### PR TITLE
Revised parsing of ICON-logfiles

### PR DESCRIPTION
**Pull request by jonasjucker**

### CODE REVIEW

**Comment by mjaehn** [comment]
The script now works again on Piz Daint assuming the `--no_sys_report` option is used. Is there anything else we need to do within this PR?

File: create_scaling_table_per_exp.py [hunk 1]
```diff
@@ -241,24 +243,19 @@ def get_wallclock_Nnodes_gen_daint(filename, string_sys_report="Elapsed"):
...
+                    # infer nnodes from MPI-procs in ICON output
+                    nodes_line = grep("mo_mpi::start_mpi ICON: Globally run on",filename)["line"][0]
+                    nnodes=int(nodes_line.split(' ')[6])
 
-                    # Euler login nodes
-                    elif 'eu-login' in hostname:
-                        nodes_line = grep("mo_mpi::start_mpi ICON: Globally run on",filename)["line"][0]
-                        nnodes=int(nodes_line.split(' ')[6])
+                    nnodes = nnodes // args.cpu_per_node
+                    print(nnodes)
```
Comment (jonasjucker) [inline create_scaling_table_per_exp.py:251]: this line can be removed right?

**Comment by jonasjucker** [comment]
Hm good question.

I'm just thinking if it makes sense to reimplement the job-summary parsing from the slurm output for Daint.
This is a model independent solution,altough I think we are the only users of the scalings_analysis tools.

**Comment by jonasjucker** [comment]
On the other hand it makes no sense to maintain two different ways of retrieving the timings...

**Comment by mjaehn** [comment]
So you mean in the case of Piz Daint to use this part of the log file:
```
Job information (1/2) 
-----------------------------------------------------------------------------------------------------
             Submit            Eligible               Start                 End    Elapsed Time limit 
------------------- ------------------- ------------------- ------------------- ---------- ----------
2021-04-12T16:17:24 2021-04-12T16:17:24 2021-04-12T16:24:59 2021-04-12T17:30:07   01:05:08 1-00:00:00
-----------------------------------------------------------------------------------------------------
```
Yes this would be model independent, but platform dependent. I would rather prefer to keep it like it is now. Also because if the batch job summary report changes again, we would have to re-adapt as well.

**Comment by jonasjucker** [comment]
Yes exactly that.
I agree with you. 
You can merge it from my point of view

File: create_scaling_table_per_exp.py (hunk 1 above)
Comment (mjaehn) [reply create_scaling_table_per_exp.py]: Yes, I removed it.

**Pull request merged**